### DISCUSSION TITLE
NocoDB Cloud [May 10]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5650] [Discord Threads]
> 🙋‍♂️ **0xhidden**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### Work in progress. We will keep you posted on this soon. ❤️  
#  
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### We would love to understand your use case. Will DM you.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1093956292286099496)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
